public/manifest.json

```diff
@@ -4,7 +4,7 @@
   "icons": [
     {
       "src": "favicon.ico",
-      "sizes": "64x64 32x32 24x24 16x16",
```

Comment:
Chrome DevTools complained about the file size being 48x48 when it's listed as these four different sizes.